config/initializers/assets.rb

```diff
@@ -48,5 +48,6 @@
   'user_tags.css.scss',
   'wiki.css',
   'horsey.js',
-  'atwho_autocomplete.js'
+  'atwho_autocomplete.js',
+  'remote_link_handler.js'
```

Comment:
Actually do you think we could include this only on that one page, using `<%= javascript_include_tag "remote_link_handler" %>` (i think that's the syntax, you may need to check on other pages to confirm this)? And perhaps a different name would be best, like `async_tag_subscriptions.js`?